cpp/models/ode_secirvvs/parameters_io.h

```diff
@@ -1434,11 +962,12 @@ IOResult<void> read_input_data_county(std::vector<Model>& model, Date date, cons
         // (This only represents the vectorization of the previous function over all simulation days...)
         log_warning("Exporting time series of extrapolated real data. This may take some minutes. "
                     "For simulation runs over the same time period, deactivate it.");
-        BOOST_OUTCOME_TRY(
-            export_input_data_county_timeseries(model, dir, county, date, scaling_factor_inf, scaling_factor_icu,
-                                                num_days, path_join(dir, "pydata/Germany", "county_divi_ma7.json"),
-                                                path_join(dir, "pydata/Germany", "cases_all_county_age_ma7.json"),
-                                                path_join(dir, "pydata/Germany", "county_current_population.json")));
+        BOOST_OUTCOME_TRY(export_input_data_county_timeseries(
+            model, dir, county, date, scaling_factor_inf, scaling_factor_icu, num_days,
+            path_join(dir, "pydata/Germany", "county_divi_ma7.json"),
```

Comment:
As in the other file: can we completely combine these two functions with one option "export" or something like this?

Reply:
see above for the secir case

Reply:
Yes, ok